Sources/TuistLoader/Models+ManifestMappers/CopyFilesAction+ManifestMapper.swift

```diff
@@ -42,7 +43,7 @@ extension TuistGraph.CopyFilesAction {
             }
         }
 
-        if !invalidResourceGlobs.isEmpty {
+        if !isExternal, !invalidResourceGlobs.isEmpty {
```

Comment:
I think this should be moved to a linter instead (and then access the `isExternal` in the linter instead of passing it through mappers)

Reply:
I think it makes sense, but I would say that change can be done independently from this fix.
If you have time, feel free to change as you wish and push on this branch, otherwise I think this can be a good shorter term fix